evap/staff/views.py

```diff
@@ -1706,7 +1706,25 @@ def template_edit(request, template_id):
         messages.success(request, _("Successfully updated template."))
         return redirect('staff:index')
 
-    return render(request, "staff_template_form.html", dict(form=form, template=template))
+    available_variables = [
+        "contact_email",
+        "page_url",
+        "login_url",        # only if they need it
+        "user"
+    ]
+
+    if template.name == EmailTemplate.STUDENT_REMINDER:
+        available_variables += ["first_due_in_days", "due_evaluations"]
+    elif template.name == EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR or template.name == EmailTemplate.PUBLISHING_NOTICE_CONTRIBUTOR or template.name == EmailTemplate.EDITOR_REVIEW_NOTICE or template.name == EmailTemplate.EDITOR_REVIEW_REMINDER:
+        available_variables += ["evaluations"]
+    elif template.name == EmailTemplate.EVALUATION_STARTED:
+        available_variables += ["evaluations", "due_evaluations"]
+    elif template.name == EmailTemplate.DIRECT_DELEGATION:
+        available_variables += ["evaluation", "delegate_user"]
+
+    available_variables = list(map(lambda s: "{{ " + s + " }}", available_variables))
```

Comment:
I guess the pythonic way would be to use a list comprehension, but I see that the excessive escaping seems weird (however, it has something aesthetical, too :D)
```python
available_variables = [f"{{{{ {var} }}}}" for var in available_variables]
```

Reply:
I think the escaping might confuse readers. I will use a list comprehension instead of mappping.